nestor_api/api/public_routes/heartbeat.py

```diff
@@ -0,0 +1,10 @@
+"""Define the heartbeat route."""
+from flask import Blueprint, jsonify
+
+blueprint = Blueprint("heartbeat", __name__)
+
+
+@blueprint.route("/heartbeat")
+def heartbeat():
+    """A heartbeat route for monitoring purpose (no auth!)."""
+    return jsonify({"state": "up"})
```

Comment:
I think we should rather just respond with a 200 without body.

Reply:
Then a 204, right?